macros/is_databricks_sql_warehouse.sql

```diff
@@ -0,0 +1,15 @@
+{% macro is_databricks_sql_warehouse(target) %}
```

Comment:
since this is currently in use in the [Fivetran Platform package](https://github.com/fivetran/dbt_fivetran_log/blob/main/macros/is_databricks_sql_warehouse.sql), should we adjust the name to `fivetran_is_databricks_sql_warehouse` to not cause any conflicts? and then we can update the reference in Fivetran Platform to use this version of the macro (and delete the old one)

otherwise i think we'll need to add an adapter.dispatch line

@fivetran-catfritz

Reply:
YES! Let's rename to be safe and consistent

Reply:
Updated.